services/QuillLMS/engines/evidence/spec/services/evidence/gen_ai_record_builder_spec.rb

```diff
@@ -33,11 +33,13 @@ module Evidence
         expect(gen_ai_activity.but_text).to eq(relevant_texts[:but_text])
       end
 
-      it 'should create a stem vault for each prompt' do
+      it 'should create a stem vault and relevant text for each prompt' do
         prompts.each do |prompt|
           stem_vault = Evidence::Research::GenAI::StemVault.find_by(prompt_id: prompt.id)
           expect(stem_vault).not_to be_nil
```

Comment:
I think you can get rid of line 39 since that failure will manifest on line 40 with the `stem_vault.stem` call.